app/Atro/Listeners/Metadata.php

```diff
@@ -89,17 +89,11 @@ protected function prepareExtensibleEnum(array &$data): void
             }
 
             foreach ($entityDefs['fields'] as $field => $fieldDefs) {
-                if (!empty($fieldDefs['type']) && in_array($fieldDefs['type'], ['extensibleEnum', 'extensibleMultiEnum', 'measure']) && empty($fieldDefs['view'])) {
+                $dropdownTypes = ['extensibleEnum', 'extensibleMultiEnum', 'link', 'linkMultiple', 'measure'];
+                if (!empty($fieldDefs['type']) && in_array($fieldDefs['type'], $dropdownTypes) && empty($fieldDefs['view'])) {
                     if (!empty($fieldDefs['dropdown'])) {
-                        if ($fieldDefs['type'] == 'extensibleMultiEnum') {
-                            $data['entityDefs'][$entityType]['fields'][$field]['view'] = 'views/fields/extensible-multi-enum-dropdown';
-                        } else {
-                            if ($fieldDefs['type'] == 'extensibleEnum') {
-                                $data['entityDefs'][$entityType]['fields'][$field]['view'] = 'views/fields/extensible-enum-dropdown';
-                            } else {
-                                $data['entityDefs'][$entityType]['fields'][$field]['view'] = 'views/fields/measure-dropdown';
-                            }
-                        }
+                        $viewType = str_replace('_', '-', Util::toUnderScore($fieldDefs['type']));
```

Comment:
Please write a normal switch case here or use a special function. Your code needs to be easie for debug.